test/rules/trailing-comma/singleline-never/test.ts.lint

```diff
@@ -114,7 +114,6 @@ var [ddOne, ddTwo] = dd;
 var a: [string, number] = null;
 
 var a: [string, number,] = null;
-                      ~          [Unnecessary trailing comma]
```

Comment:
Actually the trailing comma in tuply *types* was never valid:
http://www.typescriptlang.org/play/#src=var%20a%3A%20%5Bstring%2C%20number%2C%5D%20%3D%20null%3B
The previous implementation of this rule with option "always" even enforced the trailing comma here, which would give you compile errors. I removed the check, because typescript will show errors for unnecessary (invalid) trailing commas in this case.